okex-client/src/client/mod.rs

```diff
@@ -214,7 +215,9 @@ impl OkexClient {
         let trading_balance = Self::extract_response_data::<TradingBalanceData>(response).await?;
 
         Ok(AvailableBalance {
-            amt_in_btc: trading_balance.details[0].cash_bal,
+            free_amt_in_btc: trading_balance.details[0].avail_eq,
```

Comment:
In the event that the exchange returns an empty `trading_balance.details` array, it is helpful to check beforehand and to handle the error if that is the case. Something similar to this:
```
let trading_balance_details = trading_balance
            .details
            .first()
            .ok_or(OkexClientError::EmptyTradingBalanceData)?;

Ok(AvailableBalance {
    free_amt_in_btc: trading_balance_details.avail_eq,
    used_amt_in_btc: trading_balance_details.frozen_bal,
    total_amt_in_btc: trading_balance_details.eq,
})
```

Reply:
Yeah good point